AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.ts

```diff
@@ -243,72 +240,96 @@ export class SummaryComponent implements OnInit, OnDestroy {
                     caseName: this.hearing.cases[0].name,
                     caseNumber: this.hearing.cases[0].number
                 });
-                const hearingDetailsResponse = await this.hearingService.saveHearing(this.hearing);
 
-                if (hearingDetailsResponse.status === BookingStatus.Failed.toString()) {
-                    this.hearing.hearing_id = hearingDetailsResponse.id;
-                    this.setError(`Failed to book new hearing for ${hearingDetailsResponse.created_by} `);
-                    return;
-                }
+                const hearingDetailsResponse = await this.hearingService.saveHearing(this.hearing);
 
-                if (this.hearing.multiDays) {
-                    this.logger.info(`${this.loggerPrefix} Hearing is multi-day`, {
-                        hearingId: hearingDetailsResponse.id,
-                        caseName: this.hearing.cases[0].name,
-                        caseNumber: this.hearing.cases[0].number
+                if (this.hasJudge(hearingDetailsResponse)) {
```

Comment:
just noticed - there is already a `judgeAssigned` variable in this class. Think we can re-use that here instead of creating a new function

Reply:
this one is checking in hearing object instead my function is checking in the response object returning from the server. What should I check?

Reply:
done